pkg/liqoctl/status/localinfo_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019-2022 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package status
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pterm/pterm"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	"github.com/liqotech/liqo/apis/discovery/v1alpha1"
+	netv1alpha1 "github.com/liqotech/liqo/apis/net/v1alpha1"
+	"github.com/liqotech/liqo/pkg/liqoctl/factory"
+	"github.com/liqotech/liqo/pkg/liqoctl/output"
+	"github.com/liqotech/liqo/pkg/utils"
+	"github.com/liqotech/liqo/pkg/utils/testutil"
+)
+
+var _ = Describe("LocalInfo", func() {
+	const (
+		clusterID   = "fake"
+		clusterName = "fake"
+		namespace   = "liqo"
+	)
+
+	var (
+		clientBuilder   fake.ClientBuilder
+		lic             *LocalInfoChecker
+		ctx             = context.Background()
+		clusterIdentity v1alpha1.ClusterIdentity
+		options         = Options{Factory: factory.NewForLocal()}
+	)
+
+	BeforeEach(func() {
+		ctx = context.Background()
+		_ = netv1alpha1.AddToScheme(scheme.Scheme)
+		clientBuilder = *fake.NewClientBuilder().WithScheme(scheme.Scheme)
+		pterm.DisableStyling()
+	})
+
+	Context("Creating a new LocalInfoChecker", func() {
+		JustBeforeEach(func() {
+			lic = newLocalInfoChecker(&options)
+		})
+		It("should return a valid LocalInfoChecker", func() {
+			licTest := &LocalInfoChecker{
+				options:          &options,
+				collectionErrors: nil,
+				localInfoSection: output.NewRootSection(),
+			}
+			Expect(*lic).To(Equal(*licTest))
+		})
+	})
+	Context("Getting a local cluster identity", func() {
+		BeforeEach(func() {
+			clientBuilder.WithObjects(testutil.FakeClusterIDConfigMap(namespace, clusterID, clusterName))
+		})
+		JustBeforeEach(func() {
+			clientBuilder.Build()
+			clusterIdentity, _ = utils.GetClusterIdentityWithControllerClient(ctx, clientBuilder.Build(), namespace)
+		})
+		It("should return a valid cluster identity", func() {
+			Expect(clusterIdentity.ClusterID).To(Equal(clusterID))
+			Expect(clusterIdentity.ClusterName).To(Equal(clusterName))
+		})
+	})
```

Comment:
Unclear to me what you are testing here. The function is not defined in this package.